Makefile

```diff
@@ -47,3 +47,17 @@ clippy:
 test:
 	@cargo clean -p postgre-client -p rocks-db -p interface
 	@cargo test --features integration_tests
+
+# Ensure git-cliff is installed
+ensure-git-cliff:
+	@which git-cliff > /dev/null || cargo install git-cliff
+
+# Generate a changelog using git-cliff
+changelog:
+	@git-cliff --output CHANGELOG.md
+
+# Prepare a release (meant to be run locally)
+release:
+	@echo "Preparing release $(VERSION)"
+	@echo "This will trigger the GitHub Action to prepare a release"
+	@gh workflow run release-prepare.yml -f version=$(VERSION)
```

Comment:
i'd also add something like `if [[ -z $VERSION ]]; then echo "Please set the VERSION environment variable to the version you wish to release" && exit 1; fi`. that really asks for its own script though!